vulcanai2/tests/models/test_layers.py

```diff
@@ -48,6 +48,27 @@ def test_init(self, denseunit):
                               nn.ReLU, modules.dropout._DropoutNd))
             assert callable(unit[1])
     
+    @pytest.fixture
+    def test_denseunit_parameters(self):
+        """Create a dictionary with parameters that 
+        aren't DenseUnit parameters"""
+        return dict(
+                    in_channels=10,
+                    out_features=10,
+                    initializer=nn.init.xavier_uniform_,
+                    bias_init=0.1,
+                    norm='batch',
+                    activation=nn.ReLU(),
+                    dropout=0.5
+                )
+    
+    def test_create_denseunit(self, test_denseunit_parameters):
+        """Check if passing wrong parameters raises TypeError"""
+        with pytest.raises(TypeError) as e:
+            DenseUnit(**test_denseunit_parameters)
+            assert e.message == "__init__() got an unexpected \
```

Comment:
this assertion should be outside of the `with` statement

Reply:
👍